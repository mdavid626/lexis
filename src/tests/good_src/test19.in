10 10 20 10 50 50 7409 "dsadasdasd"
20.0 "afdsfdsfsd\"fsdfsd"
"dasdsadasdas" sdadasdsadas
10 20 "abcd"

