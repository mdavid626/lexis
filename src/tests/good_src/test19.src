/' Program 19: Input '/

scope
dim i as integer
dim d as double
dim s as string

input; i
input; d
input; s
print i; d; s; "\n";
input; i, d, s
print i; d; s; "\n";
input; i, d, s
print i; d; s; "\n";
end scope
