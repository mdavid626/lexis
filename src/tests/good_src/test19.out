? ? ?  10 1020 10 50 50 7409 "dsadasdasd"
?  20 0dasdsadasdas
?  0 1020 "abcd"
